proto/workflow.proto

```diff
@@ -169,8 +172,11 @@ message ExecuteWorkflowResponse {
   // The response context.
   context.ResponseContext response_context = 1;
 
+  // TODO(Maggie): Deprecate this field and clean up references
   // The invocation ID of the workflow execution.
   string invocation_id = 2;
+
+  map<string, string> action_name_to_invocation_id = 3;
```

Comment:
Can we use something like `repeated WorkflowActionInvocation invocations = 3;` and then have `message WorkflowActionInvocation` with invocation_id and action_name fields? That way if we need to attach more metadata to each started action, we don't need a breaking API change. (e.g. maybe we'll want to add a `status` field which contains a gRPC status for any actions that failed to start?)

Reply:
Done